provider/pkg/await/await.go

```
@@ -276,6 +276,7 @@ func Creation(c CreateConfig) (*unstructured.Unstructured, error) {
 				if waitErr != nil {
 					return nil, waitErr
 				}
+				_ = clearStatus(c.Context, c.Host, c.URN)
```

Nit: what I've seen at Pulumi before for intentionally ignoring errors is:

```
err := clearStatus(...)
contract.IgnoreError(err)
```